nailgun/entities.py

```diff
@@ -2000,6 +2014,53 @@ def update_payload(self, fields=None):
         return {u'hostgroup': super(HostGroup, self).update_payload(fields)}
 
 
+class HostPackage(Entity):
+    """A representation of a Host Package entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        if _get_version(server_config) < Version('6.2'):
+            raise NotImplementedError(
```

Comment:
You mean [this](https://github.com/SatelliteQE/nailgun/pull/276/files/0f714442b656ac44bbb5da476bd8d407b35a0df1#diff-3822710a997c3f2f7c3e93e884af69caR4183) warning? :)